drgn/helpers/linux/sched.py

```diff
@@ -83,3 +86,17 @@ def task_state_to_char(task: Object) -> str:
         return "I"
     else:
         return char
+
+
+def avenrun(prog: Program) -> Tuple[float, float, float]:
```

Comment:
How about we call this `loadavg`, since that name will be more familiar to users than the kernel-internal `avenrun` variable:

```suggestion
def loadavg(prog: Program) -> Tuple[float, float, float]:
```

(Sorry, I should've mentioned this on the last version.)